src/studio/src/designer/frontend/packages/schema-editor/src/components/Editor.tsx

```diff
@@ -174,34 +179,39 @@ export const Editor = (props: IEditorProps) => {
         </section>
         {schema ? (
           <div id='schema-editor' className={classes.editor}>
-            <TabContext value={tabIndex}>
+            <TabContext value={selectedTab}>
               <AppBar
                 position='static' color='default'
                 className={classes.appBar}
               >
                 <TabList
-                  onChange={(e, v) => setTabIndex(v)}
+                  onChange={(_x, value) => dispatch(setSelectedTab({ selectedTab: value }))}
```

Comment:
Why inline this and not inline the functions at line 149 and 153?

Reply:
Laziness... Fixed.